docs/pages/database-access/guides/redis.mdx

```diff
@@ -0,0 +1,350 @@
+---
+title: Database Access with Redis
+description: How to configure Teleport Database Access with Redis.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="9.0"
+>
+  Database access for Redis is available starting from Teleport `9.0`.
+</Details>
+
+This guide will help you to:
+
+- Install and configure Teleport.
+- Configure mutual TLS authentication between Teleport and your Redis database.
```

Comment:
I only have some brief firsthand experience with Redis, but I usually see it described as an in-memory data store rather than a database. Does "database" refer to a specific configuration of Redis, or can this guide also apply to Redis when configured as a cache (i.e. with `maxmemory` and  eviction policies set), or even other possibilities like Redis streams?

In other words, would it make more sense to just say "Redis" or "your Redis deployment," rather than "your Redis database"?

Reply:
I think that "Redis" makes sense in most places. I was thinking about using "Redis instance" as this is confusing when referring to Redis in cluster.